src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/RemoveResourceDesignerStep.cs

```diff
@@ -0,0 +1,223 @@
+using Mono.Cecil;
+using Mono.Linker;
+using Mono.Linker.Steps;
+using System;
+using System.Linq;
+using Xamarin.Android.Tasks;
+using System.Collections.Generic;
+using Mono.Cecil.Cil;
+using System.Text.RegularExpressions;
+
+namespace MonoDroid.Tuner
+{
+	public class AndroidLinkConfiguration {
+		public IList<AssemblyDefinition> Assemblies => new List<AssemblyDefinition> ();
+
+		static IDictionary<LinkContext, AndroidLinkConfiguration> configurations = new Dictionary<LinkContext, AndroidLinkConfiguration> ();
+
+		public static AndroidLinkConfiguration GetInstance (LinkContext context)
+		{
+			if (!configurations.TryGetValue (context, out AndroidLinkConfiguration config)) {
+				configurations.Add (context, new AndroidLinkConfiguration ());
+			}
+			return config;
+		}
+	}
+	public class GetAssembliesStep : BaseStep
```

Comment:
Please put these classes in `GetAssembliesStep.cs` and `AndroidLinkConfiguration.cs`.

Reply:
Yup, I know this was just me testing stuff